ui/component/notification/index.js

```diff
@@ -1,8 +1,10 @@
 import { connect } from 'react-redux';
```

Comment:
After leaving a comment reply directly from the notification component, the reply is not immediately visible. This will be confusing, and I think a toast wouldn't cut it either.  I understand it is troublesome to place the reply component there with current code.

Reply:
[![image.png](https://i.postimg.cc/7LfNhJVJ/image.png)](https://postimg.cc/FfXSnRwh)

like this?